lib/service_manager.dart

```diff
@@ -423,8 +450,11 @@ class ServiceExtensionManager {
 
   /// Sets the state for a service extension and makes the call to the VMService.
   Future<void> setServiceExtensionState(
-      String name, bool enabled, dynamic value) async {
-    await _callServiceExtension(name, value);
+      String name, bool enabled, dynamic value,
+      {bool callExtension = true}) async {
```

Comment:
use a trailing comma after `true= to get the flutter style code formatting for this method.

Reply:
done